### PR TITLE
Pylint fixes for jenkins.build, .job, and .deduper

### PR DESCRIPTION
**Pull request by benpatterson**
@jzoldak @estute 

I had some extra cycles with a low internet connection so did some pylint fixes. This gets through about 1/3 of the ones in the jenkins/ directory. PTAL.

### CODE REVIEW

File: jenkins/deduper.py [hunk 1]
```diff
@@ -143,12 +143,12 @@ def stop_duplicates(self, build_data):
                 # Now for each author, stop all but the most
                 # recent build.
                 for build_list in pr_builds.itervalues():
-                    output = self.stop_all_but_most_recent(pr, build_list)
+                    output = self.stop_all_but_most_recent(pull_request, build_list)
```
Comment (jzoldak) [inline jenkins/deduper.py]: oh snaps, now this line's too long.

$ pep8 .
./jenkins/deduper.py:146:80: E501 line too long (84 > 79 characters)

File: jenkins/build.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class Build(dict):
     :Args:
         build (dict): build data from Jenkins
     """
-    def __init__(self, build):
+    def __init__(self, build):  # pylint: disable=super-init-not-called
```
Comment (jzoldak) [inline jenkins/build.py]: Shouldn't we call the super init?

**Comment by benpatterson** [comment]
@jzoldak updated based on your comments.

**Comment by benpatterson** [comment]
Also @estute

**Comment by estute** [comment]
👍

**Comment by benpatterson** [comment]
squashing...

**Comment by jzoldak** [comment]
👍

**Pull request merged**